src/components/Admin/UserFrom.tsx

```diff
@@ -0,0 +1,346 @@
+import React, { FC, useState } from 'react';
+import { Prompt, useHistory, useParams } from 'react-router-dom';
+import { Alert, Button, Col, Form, FormControl } from 'react-bootstrap';
+import generator from 'generate-password';
+import { Formik, FieldArray } from 'formik';
+import * as yup from 'yup';
+/*lib imports end*/
+
+import InputWithCopy from './InputWithCopy';
+/*components imports end*/
+
+import gql from 'graphql-tag';
```

Comment:
This is lib import. 
Please import from '@apollo/client';

`import { gql } from '@apollo/client';`